### PR TITLE
[Travis] Add testing with PHP 7.2

### PR DESCRIPTION
**Pull request by andrerom**
With PHP 7.2 soon in stable adding some testing for it, if it passes w/o notable changes it can be added to 6.11, otherwise master.

__Side: Removing the commented out hhvm section as most libraries we use have dropped support for it after PHP 7 came out and HHVM lost steam and focused more on Hack then PHP support._

### CODE REVIEW

**Comment by andrerom** [comment]
Afaik it should pass now, so sending for review. @yannickroger given we are in beta I assume it's fine this goes into 6.11 :)

File: .travis.yml [hunk 1]
```diff
@@ -42,17 +42,15 @@ matrix:
     - php: 7.0
       env: BEHAT_OPTS="--profile=rest --tags=~@broken --suite=fullJson" RUN_INSTALL=1 SYMFONY_ENV=behat
     - php: 7.0
-      env: ELASTICSEARCH_VERSION="1.4.2" TEST_CONFIG="phpunit-integration-legacy-elasticsearch.xml" SYMFONY_VERSION="~2.8.0"
-    - php: 7.0
       env: SOLR_VERSION="6.4.2" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CUSTOM_CACHE_POOL="singleredis" CORES_SETUP="shared" SOLR_CONFIG="vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/schema.xml vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/custom-fields-types.xml vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/language-fieldtypes.xml"
 # 7.1
     - php: 7.1
-      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="^3.2.0"
```
Comment (alongosz) [inline .travis.yml]: Given `7.2` is still beta I wouldn't drop unit testing for stable `7.1`.

Comment (andrerom) [inline .travis.yml]: True, but I try to balance the amount of jobs we need to run before we get feedback from travis, hence unit tests on lowest and highest version, and integration test variants spread across all :)

Comment (andrerom) [inline .travis.yml]: @alongosz wdyt? Most of us use 7.1 atm locally for phpunit / integration, and on top newer php versions are typically more strict. So combination of running unit tests on 5.6 and 7.2 covers 99.9% of the php version differences gotchas we encounter when coding.

Comment (alongosz) [inline .travis.yml]: Ok, you've convinced me :)

**Pull request merged**